.gitignore

```diff
@@ -32,3 +33,6 @@ yarn-error.log*
 
 # vercel
 .vercel
+.tsbuildinfo
+lib/destack/.tsbuildinfo
```

Comment:
Can we move this to `lib/destack/.gitignore`?

Reply:
`lib/destack` just has its own .gitignore to make things a bit clearer. There isn't any special configuration to it, it just works!